metrics/psutils/cpu.lua

```diff
@@ -0,0 +1,34 @@
+-- Linux is the only supported platform
+if require('ffi').os ~= 'Linux' then
+    return nil
```

Comment:
In order to preserve interface on different systems it's better to return dummy update function.

`return  { update = function() end }`

(and may be for OS detection you can simply use `jit.os` - ffi require seems to be redundant)

Reply:
done